.github/workflows/ubuntu_20.04/Dockerfile.ci

```
@@ -258,6 +258,25 @@ RUN mkdir sqlite \
     && cd .. \
     && rm -rf sqlite
 
+# Build libOpenDRIVE
+ARG OPENDRIVE_VERSION=0.5.0-gdal
+RUN . /buildscripts/bh-set-envvars.sh \
+    && if test "${OPENDRIVE_VERSION}" != ""; then ( \
```

bh-set-envvars.sh is not used in CI Dockerfiles:

```suggestion
RUN if test "${OPENDRIVE_VERSION}" != ""; then ( \
```